src-vue/src/plugins/store.ts

```diff
@@ -59,6 +63,47 @@ export const store = createStore<FlightCoreStore>({
         updateCurrentTab(state: any, newTab: Tabs) {
             state.current_tab = newTab;
         },
+        async updateGamePath(state: FlightCoreStore) {
+            // Open a selection dialog for directories
+            const selected = await open({
+                directory: true,
+                multiple: false,
+                defaultPath: await appDir(),
+            });
+            if (Array.isArray(selected)) {
+                // user selected multiple directories
+                alert("Please only select a single directory");
+            } else if (selected === null) {
+                // user cancelled the selection
+            } else {
+                // user selected a single directory
+
+                // Verify if valid Titanfall2 install location
+                let is_valid_titanfall2_install = await invoke("verify_install_location", { gamePath: selected }) as boolean;
+                if (is_valid_titanfall2_install) {
+                    state.game_path = selected;
```

Comment:
`state.install_type` needs to be also set (preferably to [`UNKNOWN`](https://github.com/GeckoEidechse/FlightCore/blob/52394a3e86373801f5285843d6931a692a619be1/src-vue/src/utils/InstallType.ts#L7)).

Otherwise this error message pops up when trying to launch the game after manually selecting the game path ^^

![screenshot](https://user-images.githubusercontent.com/40122905/194677928-0a455cd9-25cb-4bc6-ae87-85d9ecdbb56b.png)

Reply:
This shouldn't happen anymore.